src/pages/loginPage/loginPage.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import './_loginPage.scss';
+import Input from '../../components/input/input';
+import Title from '../../components/title/title';
+import Checkbox from '../../components/checkbox/checkbox';
+import Button from '../../components/button/button';
+import { Link, useNavigate } from 'react-router-dom';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { InputName } from '../../types/components/componentType';
+import { toast } from 'react-toastify';
+
+const LoginPage: React.FC = () => {
+  const navigate = useNavigate();
+
+  // State for making password visible
+  const [showPassword, setShowPassword] = useState(false);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<InputName>();
+
+  // Getting data from inputs and send it to backend
+  const loginHandler: SubmitHandler<InputName> = async (data) => {
+    try {
+      console.log(data);
+      const { username, password } = data;
+      console.log(username, password);
```

Comment:
The current code sends the username and password directly to the console. This is not a secure practice. It would be better to use a secure mechanism for sending the login credentials, such as HTTPS.